doc/metadata/README.md

```diff
@@ -0,0 +1,20 @@
+Verify in-toto's supply chain with in-toto
+===========================================
+
+**Overview**  
+This document describes how to use in-toto to verify the software supply chains for the PyPi source of in-toto v1.0.1.
+
+The basic supply chains are laid out in "in_toto_sdist.layout" and consist of two steps:
+
+1. "git-tag" -- Sign the tag using the command "git tag --sign v1.0.1 -m "v1.0.1"".
+2. "create-sdist" -- Creates source distribution (.tar.gz) using products from (1) as materials.
+
+**Notes/Discussion**  
+
+- For the bootstrapping/chicken-egg situation, we assume that the user already has an older version of the in-toto locally.
+- For the first step, we want to explicitly allow all the files to make sure we tag all the materials files to sign.
+- Since we only sign the tag in the first step, no files will be changed and there will be no products. So we only need to add the "DISALLOW *" rule for the expected products.
+- For the second step, we match all the files with the materials in the first step, because we don't have products in the first step.
+- We may want to use the in-toto-run command to create the links files and be specific for the materials and products, for example, "in-toto-run --step-name create-sdist --materials . --products in-toto.1.0.1.tar.gz --key owner --python setup.py sdist"
```

Comment:
I wonder if we'll want to `lstrip` the path to the in-toto project on the local machine where this is run.